zng/varint.go

```diff
@@ -5,7 +5,7 @@ import "math"
 // These functions are like varint but their size is known as
 // zval encoded it for them.
 
-func decodeUint(b []byte) uint64 {
+func decodeUvarint(b []byte) uint64 {
```

Comment:
I don't think we should use varint in the name of this. It already means something different in encoding/binary and zbuf.

Reply:
If you're looking for a name, maybe trimint since they trim unneeded zero bytes?

Reply:
I moved these functions to zcode and gave them a better name...